clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1409,4 +1409,54 @@ public boolean visible(String name, Map<String, Object> parsedConfig) {
         };
     }
 
+    public String toHtml() {
+        return toHtml(Collections.<String, String>emptyMap());
+    }
+
+    /**
+     * Converts this config into an HTML list that can be embedded into docs.
+     * If <code>dynamicUpdateModes</code> is non-empty, a "Dynamic Update Mode" label
+     * will be included in the config details with the value of the update mode. Default
+     * mode is "read-only".
+     * @param dynamicUpdateModes Config name -&gt; update mode mapping
+     */
+    public String toHtml(Map<String, String> dynamicUpdateModes) {
```

Comment:
Does this need to be public?

Reply:
This is called by `KafkaConfig`, see https://github.com/apache/kafka/pull/6870/files#diff-71645c36b0dca57cf3e01e06e53f5f17R261

Reply:
OK, thanks.